lib/triton_kernel_call.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include <mutex>
+#include <string>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "cuda.h"
+
+namespace jax_triton {
+
+typedef std::unordered_map<std::string, pybind11::object> asm_map_t;
```

Comment:
It is preferred to use "using":

using asm_map_t = std::unordered_map<...>;

Reply:
Done.